reports/pin/pin.qmd

```diff
@@ -76,81 +279,329 @@ assessment_pin %>%
+    `Proration Rate` = scales::percent(meta_tieback_proration_rate,
+      accuracy = 0.01
+    ),
+    `Card %` = scales::percent(meta_card_pct_total_fmv, accuracy = 0.01),
+    `Card Final` = scales::dollar(card_final)
+  )
+
+
+multi_data %>%
+  gt() %>%
+  tab_options(
+    table.width = pct(100)
   ) %>%
-  arrange(meta_year) %>%
-  select(
-    Year = meta_year,
-    Class = meta_class,
-    "Year Built" = char_yrblt,
-    "Building SF" = char_bldg_sf,
-    Beds = char_beds,
-    "Full Baths" = char_fbath,
-    "Half Baths" = char_hbath,
-    Air = char_air,
-    "Residential Type" = char_type_resd,
-    Rooms = char_rooms,
-    HIE = hie,
-    "Multicard Flag" = flag_pin_is_multicard,
-    "Missing Critical Value" = flag_char_missing_critical_value,
-  ) %>%
-  kable() %>%
-  kable_styling(
-    "striped",
-    position = "left",
-    fixed_thead = TRUE
+  tab_footnote(
+    footnote = "Percentage of total PIN building value an individual card
+      is worth",
+    locations = cells_column_labels(columns = `Card %`)
+  ) %>%
+  tab_footnote(
+    footnote = "Initial fair-market value predicted by model for each card.",
+    locations = cells_column_labels(columns = "Card Initial")
+  ) %>%
+  tab_footnote(
+    footnote = "Sum of `Card Initial.` Equal to initial card value if PIN is
+    not multicard",
+    locations = cells_column_labels(columns = "PIN Initial")
+  ) %>%
+  tab_footnote(
+    footnote = "Average of initial predictions for a card that has multiple
+    predictions since it is prorated across multiple pins. Equal to initial
+    prediction if card is not prorated",
+    locations = cells_column_labels(columns = "Card Intermediate")
+  ) %>%
+  tab_footnote(
+    footnote = "Sum of of card intermediate values",
+    locations = cells_column_labels(columns = "Card Sum")
+  ) %>%
+  tab_footnote(
+    footnote = "Card sum",
+    locations = cells_column_labels(columns = "PIN Final")
+  ) %>%
+  tab_footnote(
+    footnote = "Value of PIN from year it was last reassessed",
+    locations = cells_column_labels(columns = "PIN Far")
+  ) %>%
+  tab_footnote(
+    footnote = "Value of PIN from previous tax year",
+    locations = cells_column_labels(columns = "PIN Near")
+  ) %>%
+  tab_footnote(
+    footnote = "Percentage of a multi-PIN card that belongs to each PIN",
+    locations = cells_column_labels(columns = "Proration Rate")
+  ) %>%
+  tab_footnote(
+    footnote = "Total building value of PIN * card %",
+    locations = cells_column_labels(columns = "Card Final")
+  ) %>%
+  cols_hide(columns = c(
+    meta_pin, char_class, meta_card_num,
+    meta_card_pct_total_fmv, char_bldg_sf,
+    pred_card_initial_fmv, pred_card_final_fmv,
+    meta_complex_id, meta_1yr_pri_board_tot,
+    meta_tieback_proration_rate, prior_far_tot,
+    prior_near_tot, pred_pin_initial_fmv, pred_pin_final_fmv,
+    meta_tieback_key_pin, proration, card_intermediate,
+    pred_pin_card_sum, pred_pin_final_fmv_round
+  ))
+```
+
+
+## Flags 
```

Comment:
**suggestion:** I kind of like the flags under the left column near the picture. If they're supposed to signal something odd about the PIN then it seems important that they not be hidden away.